src/components/reusable/timepicker/timepicker.scss

```diff
@@ -0,0 +1,143 @@
+@use '../../../common/scss/global.scss';
+@use '~@kyndryl-design-system/foundation/scss/mixins/typography.scss';
+
+:host {
+  display: inline-block;
+}
+
+.label-text {
+  @include typography.type-ui-03;
+  text-transform: uppercase;
+  font-weight: 500;
+  display: block;
+  padding: 0;
+  margin-bottom: 8px;
+
+  [disabled] & {
+    color: var(--kd-color-text-disabled);
+  }
+}
+
+.required {
+  color: var(--kd-color-text-error);
+
+  [disabled] & {
+    color: var(--kd-color-text-disabled);
+  }
+}
+
+.input-wrapper {
+  position: relative;
+  display: inline-flex;
+}
+
+input {
+  @include typography.type-body-02;
+  -webkit-appearance: none;
+  appearance: none;
+  border: 1px solid var(--kd-color-border-ui);
+  border-radius: 4px;
+  height: 48px;
+  padding: 10px 16px 10px 16px;
+  width: 167px;
```

Comment:
Another fixed width lingering here.,

Reply:
in UX width of picker is 167px

Reply:
Yeah ignore that, it would only be a `min-width` if anything.

Reply:
ok will put min-width:167px; btw from when width:233px coming from ? i can see for textinput width:233px inherited from somewhere

Reply:
I'm not seeing this

Reply:
<img width="1745" alt="Screenshot 2023-08-30 at 9 04 04 PM" src="https://github.com/kyndryl-design-system/web-components/assets/140634142/75d32499-3561-43d3-ad6f-546f530afde7">

Reply:
Ah that's just the computed value, it is automatic width.

Reply:
ok.